spec/requests/api/v1/authentication_controller/oauth_signup_spec.rb

```diff
@@ -45,6 +45,20 @@
       end
     end
 
+    context "with empty email & valid provider" do
+      before do
+        post "/api/v1/oauth/signup", params: {
+          "access_token": "empty_email_token",
+          "provider": "google"
+        }, as: :json
+      end
+
+      it "return status 422 and should have jsonapi errors" do
+        expect(response).to have_http_status(422)
+        expect(response.parsed_body).to have_jsonapi_errors
```

Comment:
Can you check the exact error message, need to make end user aware that signup failed due to missing email in the linked account

Reply:
Done! @tachyons